docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+---
+version: "2.1"
+services:
+  mariadb:
+    image: lscr.io/linuxserver/mariadb:latest
+    container_name: mariadb
+    environment:
+      - PUID=501
+      - PGID=20
+      - MYSQL_ROOT_PASSWORD=Password123
+      - TZ=Europe/London
+    volumes:
+      - ./database:/config
+    ports:
+      - 4306:3306
+    restart: unless-stopped
+
+  baseball:
+    build:
+      context: .
+      dockerfile: dockerfile
+    image: baseball
+    volumes:
+      - ./baseball.sql:/App/baseball.sql
+    command: sleep 9999999
```

Comment:
This line causes the container to pause which is good if you want to enter the container and run some test commands. But to actually have it automatically run you'll want to remove this line.